js/src/views/Account/store.js

```diff
@@ -52,4 +70,27 @@ export default class Store {
   @action toggleVerificationDialog = () => {
     this.isVerificationVisible = !this.isVerificationVisible;
   }
+
+  onExport = () => {
```

Comment:
Duplicated, DRY and keep one store that is used by all the different components.